### DISCUSSION TITLE
Render of LaTeX in mobile (*.md files)

### DISCUSSION THREAD

**Discussion by andr-ll** [community/community#41372] [Mobile]
Hello!

I've been writing some documentation for my repo and found out that mobile version of app does not show valid output for LaTeX syntax.

Mobile app:
![image](https://user-images.githubusercontent.com/72939898/206879125-ae5bd384-ba57-46b2-a267-06d46632b97b.jpeg)

GitHub web:
![image](https://user-images.githubusercontent.com/72939898/206879132-d9abbf88-7371-4fb9-aae2-71a46b655a34.jpeg)

The source file is [available here](https://github.com/andr-ll/nuti/blob/master/docs/logger.md).

Is this acceptable or no?

Thanks in advance :)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.92.0 (85285908)
Device: iPhone Xs (iOS 16.1.2)
</details>

**Comment by eliperkins** [comment]
Hey @andr-ll , thanks for this feedback. Having LaTeX render in GitHub Mobile is a feature I would love to see too! I've shared these notes on our internal tracking issue about LaTeX on mobile. I'll keep you posted!

**Reply by andr-ll** [reply]
Hi @eliperkins!
Thanks for your reply, will be waiting for any news!

**Reply by shermansiu** [reply]
@eliperkins Bumping this issue a year later, as it's still a wanted feature.

**Comment by ettaboyle** [comment]
@raman-m Thank you for sharing this post - and adding it to the appropriate thread! ❤️ 

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is **strongly discouraged** and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by sinaatalay** [comment]
This would be great!

**Reply by Macaulay1234** [reply]
Thanks you

**Comment by Macaulay1234** [comment]
> Hello!
> 
> I've been writing some documentation for my repo and found out that mobile version of app does not show valid output for LaTeX syntax.
> 
> Mobile app:
> ![image](https://user-images.githubusercontent.com/72939898/206879125-ae5bd384-ba57-46b2-a267-06d46632b97b.jpeg)
> 
> GitHub web:
> ![image](https://user-images.githubusercontent.com/72939898/206879132-d9abbf88-7371-4fb9-aae2-71a46b655a34.jpeg)
> 
> The source file is [available here](https://github.com/andr-ll/nuti/blob/master/docs/logger.md).
> 
> Is this acceptable or no?
> 
> Thanks in advance :)
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.92.0 (85285908)
> Device: iPhone Xs (iOS 16.1.2)
> </details>

**Comment by Macaulay1234** [comment]
> Hey @andr-ll , thanks for this feedback. Having LaTeX render in GitHub Mobile is a feature I would love to see too! I've shared these notes on our internal tracking issue about LaTeX on mobile. I'll keep you posted!

**Comment by Macaulay1234** [comment]
> Hey @andr-ll , thanks for this feedback. Having LaTeX render in GitHub Mobile is a feature I would love to see too! I've shared these notes on our internal tracking issue about LaTeX on mobile. I'll keep you posted!

**Comment by Macaulay1234** [comment]
> The same issue. Inline LaTeX expression is not processed in GitHub Mobile Android app.
> But this is very simple expression.
> 
> Attention! Large picture! 😄
> <details>
>   <summary>Click here</summary>
> 
> ![image](https://github.com/community/community/assets/10501504/72b21427-9f2a-4636-85c4-7e64a5ebde2c)
> </details>
> 
> #### Markdown code
> ```md
> ### $\color{red}{There\ are\ 2\ failed\ tests\ in\ the\ build!}$
> 
> $\color{red}{There\ are\ 2\ failed\ tests\ in\ the\ build!}$
> 
> **Build**: [1422](https://circleci.com/gh/ThreeMammals/Ocelot/1422)
> ```
> 
> #### Direct Link
> - [1372](https://github.com/ThreeMammals/Ocelot/pull/1372#pullrequestreview-1574250787)
>

**Comment by Macaulay1234** [comment]
> @raman-m Thank you for sharing this post - and adding it to the appropriate thread! ❤️ 
> 
> Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is **strongly discouraged** and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.
> 
> This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.
> 
> If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."
> 
> Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.
> 
> Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by d-kleine** [comment]
As the issue that GitHub app still does not render LaTeX markdown is stale for several years now, I have opened a support ticket (#3019480; can only be seen by the GitHub Support Team) some weeks ago on this. I consider this as **bug** but not a feature request as LaTeX is fundamental to GitHub's utility as a platform for developers and researchers. Since [GitHub introduced LaTeX support for Markdown on May 19, 2022](https://github.blog/news-insights/product-news/math-support-in-markdown/), this "feature" has never worked before in the GitHub app and affects both Android as well as iOS users.

**Sources Where This Issue Has Been Reported Before:**

- #54076
- #41608 
- #128209 
- #36915 
- #41608 
- #41372 
- #18128 
- [Reddit Discussion: LaTeX matrix not rendering correctly on GitHub Markdown](https://www.reddit.com/r/github/comments/yen375/latex_matrix_not_rendering_correctly_on_github/)
... and many more.

❗ **To all users affected by this issue**: Please make sure to to give a *thumbs up* 👍🏻  / *arrow up* ⬆️  vote in the @andr-ll's original report to this issue so that the product managers can see the demand for this feature. Highlighting your support is **crucial** because it helps prioritize the development of LaTeX rendering in the GitHub Mobile app, ensuring a consistent user experience across platforms and addressing a significant need for those who rely on LaTeX for their work.

**Comment by markcst** [comment]
It's almost 2025 and Microsoft still hasn't done this crap?
And move along, you just need to implement Latex rendering in an app, not go to the moon!

**Reply by d-kleine** [reply]
GitDev devs consider this a feature request, not a bug, for the app. That's why you need to upvote the above original post from @andr-ll to show the product managers that it is important (even sometimes critical) for users like us.

**Reply by XPD-Kasun** [reply]
Why the devs cant make this work? I think they are using a webview and just adding mathjax can at least do some justice.

**Comment by XPD-Kasun** [comment]
Referring [this issue](https://github.com/orgs/community/discussions/144688) also. Same as @andr-ll but the same issue persist in Android app too.
